packages/components/psammead-media-player/src/MediaMessage/index.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import MediaMessage from './index';
+import '@testing-library/jest-dom/extend-expect';
+
+it('should display the media message', () => {
+  const { getByText } = render(
+    <MediaMessage
+      service="news"
+      noJsMessage="Please enable Javascript or try a different browser"
+      placeholderSrc="http://foo.bar/placeholder.png"
+      placeholderSrcset="http://foo.bar/placeholder.png"
+    />,
+  );
+  const mediaMessage = getByText(
+    'Please enable Javascript or try a different browser',
```

Comment:
What HTML element will the message be wrapped in @rebeccarfmcginnbbc Just checking if this also follows the screen reader UX? https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5e2b162c84cd309968 Also any plans for an accessibility swarm on this? Maybe it's been done? Just checking...